docs/installation.md

```diff
@@ -2,50 +2,73 @@
 title: Installation
 ---
 
-Reason comes by default in [BuckleScript](https://bucklescript.github.io/), a compiler that turns Reason code into JavaScript code.
+> **Important:** In case you are looking for BuckleScript + Reason instructions, please note that BuckleScript has rebranded to ReScript and created its own syntax (it's very similar to Reason that is targeted more towards JS development needs). Please refer to the [ReScript website](https://rescript-lang.org) for more infos.
```

Comment:
I don't think there needs to be an explanation of what BuckleScript/ReScript is here. A simple "If you are looking for ReScript (formerly BuckleScript) documentation please refer to the [ReScript website]."

Reply:
Good point. Changed to your suggestion